bench/ms/ms.mjs

```diff
@@ -0,0 +1,81 @@
+import { ms } from "ms";
+import { bench, group, run } from "../runner.mjs";
+
+const stringInputs = ["1s", "1m", "1h", "1d", "1w", "1y", "2 days", "10h", "2.5 hrs", "1.5h", "100ms"];
+const numberInputs = [1000, 60000, 3600000, 86400000, 604800000];
```

Comment:
Why are these two separate arrays?

```typescript
const inputs = [{ string: "1s", numeric: 1000}, ...];
const inputs = [["1s", 1000], ...];
```

either would be better

Reply:
They are kinda testing different things

Reply:
oh im stupid sorry